packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md

```diff
@@ -0,0 +1,440 @@
+  --url https://${yourOktaDomain}/api/v1/authorizationServers/{AuthorizationServerId}/policies/{policyId}/rules/{ruleId} \
+  --header 'Content-Type: application/json' \
+  --header 'Accept: application/json' \
+  --header 'Authorization: SSWS {apiKey}'
+```
+
+### Add the token exchange grant type to the policy
+
+Next, configure the token exchange grant type for the client that is using the API. You need to update the `grantTypes` parameter by adding the value `urn:ietf:params:oauth:grant-type:token-exchange` so that the token exchange is an allowed grant type in the policy rule.
+
+> **Note:** The `*` value in `scopes` means any scope is valid here.
+
+**Update example**
+
+```json
+curl --location --request PUT
+--url https://${yourOktaDomain}/api/v1/authorizationServers/{AuthorizationServerId}/policies/{policyId}/rules/{ruleId} \
+--header 'Content-Type: application/json' \
+--header 'Accept: application/json' \
+--header 'Authorization: SSWS <apiKey>' \
+-d '{
+	"id": "0pr3erdfwAnLPICrw0g4",
+	"status": "ACTIVE",
+	"name": "allow token exchange",
+	"priority": 1,
+	"created": "2021-04-14T17:47:42.000Z",
+	"lastUpdated": "2021-04-14T17:47:42.000Z",
+	"system": false,
+	"conditions": {
+    	"people": {
+        	"users": {
+            	"include": [],
+            	"exclude": []
+        	},
+        	"groups": {
+            	"include": [
+                	"EVERYONE"
+            	],
+            	"exclude": []
+        	}
+    	},
+    	"grantTypes": {
+        	"include": [
+            	"implicit",
+            	"password",
+				"client_credentials",
+            	"authorization_code",
+            	"urn:ietf:params:oauth:grant-type:token-exchange"
+        	]
+    	},
+    	"scopes": {
+        	"include": [
+            	"*"
+        	]
+    	}
+	},
+	"actions": {
+    	"token": {
+        	"accessTokenLifetimeMinutes": 60,
+        	"refreshTokenLifetimeMinutes": 0,
+        	"refreshTokenWindowMinutes": 10080
+    	}
+	},
+	"type": "RESOURCE_ACCESS"
+}'
+```
+
+## Configure your client to participate in Native SSO
+
+This is applicable for all clients participating in Native SSO (for example, client 1 and client 2). In this request, use the client ID for the client that you created in the "Set up the application" section. Copy the response that is returned from this request for use in the next step.
+
+**Request example**
+
+```bash
+  curl --request GET \
+  --url https://${yourOktaDomain}/oauth2/v1/clients/{clientId}' \
+  --header 'Content-Type: application/json' \
+  --header 'Accept: application/json' \
+  --header 'Authorization: SSWS <apiKey>'
+```
+
+### Update the client with the token exchange grant
+
+In this request, update the client with the token exchange grant. Use the response from the last step to create your UPDATE request. You need to update the `grantTypes` parameter by adding the value `urn:ietf:params:oauth:grant-type:token-exchange` so that the token exchange is an allowed grant type for the client.
```

Comment:
```suggestion
In this request, update the client with the token exchange grant. Use the response from the last step to create your UPDATE request. You need to update the `grant_types` parameter by adding the value `urn:ietf:params:oauth:grant-type:token-exchange` so that the token exchange is an allowed grant type for the client.
```